Makefile

```diff
@@ -6,8 +6,9 @@ NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
 # but this can be overridden when calling make, e.g.
 # CONTAINER_ENGINE=podman make container-image
 CONTAINER_ENGINE ?= docker
+IMAGE_REGISTRY ?= gcr.io/k8s-staging-sig-docs
```

Comment:
If someone builds this locally, what tag should the image have?
(maybe add a suffix if the build is _not_ via cloud-build?)

Reply:
The tag will be something like `gcr.io/k8s-staging-sig-docs/k8s-website-hugo:v0.74.3-ce834e382c09`. I don't have strong opinion about having a suffix for cloudbuild.

Reply:
Can we use dirty commit hash?

Reply:
> dirty commit hash

(no objection)